edk2-rockchip/Platform/Firefly/ROC-RK3568-PC/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c

```diff
@@ -1000,7 +1000,7 @@ PhyMemArrayInfoUpdateSmbiosType16 (
  //  - Type 17 VolatileSize in Bytes
  //
 
-  mMemDevInfoType17.Size = PcdGet64(PcdSystemMemorySize) / (1024 * 1024);;
+  mMemDevInfoType17.Size = mMemorySize / (1024 * 1024);
```

Comment:
Oops good catch. Can you submit this in a separate change?

Reply:
Yes, I will separate everything into separate pull requests